src/dotnet/commands/dotnet-clean/Program.cs

```diff
@@ -17,20 +17,19 @@ public static int Run(string[] args)
             CommandLineApplication app = new CommandLineApplication(throwOnUnexpectedArg: false)
             {
                 Name = "dotnet clean",
-                FullName = ".NET Clean Command",
-                Description = "Command to clean previously generated build outputs.",
+                FullName = LocalizableStrings.AppFullName,
+                Description = LocalizableStrings.AppDescription,
                 AllowArgumentSeparator = true,
                 ArgumentSeparatorHelpText = HelpMessageStrings.MSBuildAdditionalArgsHelpText
             };
             app.HelpOption("-h|--help");
 
-            CommandArgument projectArgument = app.Argument("<PROJECT>",
-                "The MSBuild project file to build. If a project file is not specified," +
-                " MSBuild searches the current working directory for a file that has a file extension that ends in `proj` and uses that file.");
+            CommandArgument projectArgument = app.Argument($"<{LocalizableStrings.CmdArgProject}>",
+                LocalizableStrings.CmdArgProjDescription);
 
-            CommandOption outputOption = app.Option("-o|--output <OUTPUT_DIR>", "Directory in which the build outputs have been placed", CommandOptionType.SingleValue);
-            CommandOption frameworkOption = app.Option("-f|--framework <FRAMEWORK>", "Clean a specific framework", CommandOptionType.SingleValue);
-            CommandOption configurationOption = app.Option("-c|--configuration <CONFIGURATION>", "Clean a specific configuration", CommandOptionType.SingleValue);
+            CommandOption outputOption = app.Option($"-o|--output <{LocalizableStrings.CmdOutputDir}>", LocalizableStrings.CmdOutputDirDescription, CommandOptionType.SingleValue);
```

Comment:
NIT: please, keep the lines to less than 120 characters.

And when you break the lines, I would prefer that when you put parameters in a new line that each of them have their own new line. Line this:

cmd.Option(
$"...",
Localiza...,
CommandOptionType...);